### PR TITLE
Fix syntax error about `FBLPromise+Wrap.m`

### PR DESCRIPTION
**Pull request by sushichop**
This pull request fixes this issue: #164 
(I'll fix this issue since there seems to be no response about it for a while...)

Thanks in advance.

### CODE REVIEW

**Comment by sushichop** [comment]
And I checked unit test passed using Xcode🙂

File: Tests/FBLPromisesTests/FBLPromise+WrapTests.m [hunk 1]
```diff
@@ -220,13 +220,13 @@ - (void)testPromiseWrapErrorOrObjectCompletionRejectsOnValueAndErrorReturned {
 
 - (void)testPromiseWrap2ObjectsOrErrorCompletionFulfillsOnValueReturned {
   // Arrange.
-  NSArray<NSNumber *> *expectedValues = @[ @42, @13 ];
+  NSArray<NSObject *> *expectedValues = @[ @42, [NSNull null] ];
```
Comment (shoumikhin) [inline Tests/FBLPromisesTests/FBLPromise+WrapTests.m:223]: Just skip the generic arg, ie. `NSArray *expectedObjects`, which is a synonym for `NSArray<id>`

Comment (sushichop) [reply Tests/FBLPromisesTests/FBLPromise+WrapTests.m]: That makes sense.
I have updated the above.

**Comment by sushichop** [comment]
I have updated the commit. Could you check it?

File: Sources/FBLPromises/FBLPromise+Wrap.m [hunk 2]
```diff
@@ -133,7 +133,7 @@ + (instancetype)onQueue:(dispatch_queue_t)queue
                      if (error) {
                        reject(error);
                      } else {
-                       fulfill(@[ value1, value2 ]);
+                       fulfill(@[ value1 ?:[NSNull null], value2 ?: [NSNull null] ]);
```
Comment (ykjchen) [inline Sources/FBLPromises/FBLPromise+Wrap.m:136]: Could you add a single whitespace after the operator in the first array element for consistency?

```
value1 ?: [NSNull null]
```

Thanks!

**Review by ykjchen — CHANGES_REQUESTED** [review]

File: Sources/FBLPromises/FBLPromise+Wrap.m (hunk 2 above)
Comment (sushichop) [reply Sources/FBLPromises/FBLPromise+Wrap.m]: Done:)

**Comment by ykjchen** [comment]
Thanks!

**Pull request merged**